generate/baseGenerator.go

```diff
@@ -61,9 +61,11 @@ func (bg *baseGenerator) computeInitialNodesForWalletKey(key *data.WalletKey) []
 	initialNodes := make([]*sharding.InitialNode, 0, len(key.BlsKeys))
 
 	for _, blsKey := range key.BlsKeys {
+		validatorPubKey, _ := bg.validatorPubKeyConverter.Encode(blsKey.PubKeyBytes)
+		walletAddress, _ := bg.walletPubKeyConverter.Encode(key.PubKeyBytes)
```

Comment:
Can add a newline after this (formatting).

Reply:
Done.